app/Services/FileSystem/Swift/SwiftServiceProvider.php

```diff
@@ -23,6 +24,36 @@
  */
 final class SwiftServiceProvider extends ServiceProvider
 {
+    const MAX_SKIPS = 5;
+
+    private $skip_count = 0;
```

Comment:
lets save all this on redis bc production is a multinode environment

Reply:
done